packages/eslint-plugin/tests/rules/switch-exhaustiveness-check.test.ts

```diff
@@ -602,5 +602,47 @@ function test(arg: Enum): string {
         },
       ],
     },
+    {
+      code: `
+        enum Enum {
+          'a' = 1,
+          [\`key-with
+
+          new-line\`] = 2,
+        }
+
+        declare const a: Enum;
+
+        switch (a) {
+        }
+      `,
+      errors: [
+        {
+          messageId: 'switchIsNotExhaustive',
+          suggestions: [
+            {
+              messageId: 'addMissingCases',
+              output: `
+        enum Enum {
+          'a' = 1,
+          [\`key-with
+
+          new-line\`] = 2,
+        }
+
+        declare const a: Enum;
+
+        switch (a) {
+        case Enum.a: { throw new Error('Not implemented yet: Enum.a case') }
+        case Enum[\`key-with
+
+          new-line\`]: { throw new Error('Not implemented yet: Enum[\`key-with\\n\\n          new-line\`] case') }
```

Comment:
wasn't sure about this. should we throw an error with actual new lines or `\n` is fine?

Reply:
Yeah good question. It's such a rare case, I don't think it matters too much. But in general having error messages be on one line is easier to read.

Reply:
should we just strip out any spaces, tabs, and new lines?

OK... it might look weird in some cases... IDK
```JS
key = `key-with
          new-line`

error = "not implemented: 'key-withnew-line'";
```

anyway, as you said, it's a rare case that probably isn't worth the effort